File: Nodejs/csv/csv.py
# -*- coding: utf-8 -*-
archive = open('kevin.csv')
for line in archive:
    splitted = line.split(',')
    i = 0
    for word in splitted:
        splitted[i] = word.strip()
        i = i + 1

    name = splitted[0]
    age = splitted[1]
    status = splitted[2]
    print('Oi, meu nome é {}, tenho {} anos e sou {}'.format(name, age, status))

File: Nodejs/csv/csv.js
const fs = require('fs');

fs.readFile('kevin.csv', show);

function show(err, data) {
    let lines = data.toString().split('\n');

    let obj = [];
    for(line of lines) {
        let splitted = line.split(',');
        obj.push({
            name: splitted[0],
            age: splitted[1],
            status: splitted[2]
        })
    }

    for(person of obj) {

        console.log(person.name, person.age)

    }
    console.log(obj)
}